src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -252,17 +315,27 @@
     // Represents rows from stop_times.txt
     public static final GraphQLObjectType stopTimeType = newObject().name("stopTime")
             .field(MapFetcher.field("trip_id"))
-            .field(MapFetcher.field("stop_id"))
-            .field(MapFetcher.field("stop_sequence", GraphQLInt))
             .field(MapFetcher.field("arrival_time", GraphQLInt))
             .field(MapFetcher.field("departure_time", GraphQLInt))
+            .field(MapFetcher.field("stop_id"))
```

Comment:
Was there a point in reordering the existing fields?

Reply:
No reverted back.